index.html

```diff
@@ -0,0 +1,268 @@
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul  class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+        
+     </div>
+   </section>
+
+   <section class="section-3">
+     <div class="about-me">
+     <div class="me">
+       <h2> About me </h2>
+     </div>
+     <div class="p-me">
+       <p>
+       Hello I’m a software developer! I can help you build a product ,
+       feature or website Look through some of my work and
+       experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+       </p>
+     </div>
+     <div class="resume">
+       <a href="#"> <img src="image/Enable.png" alt="button">  </a>
+     </div>
+     <div class="abt-me-img">
+       <img src="image/abtme.png" alt="abtme"> 
+     </div>
```

Comment:
- [ ] Kindly reposition this image because it makes your page overflows and makes horizontal scrolling, Kindly fix this issue to match the Figma design.

![image](https://user-images.githubusercontent.com/35535130/166128121-efb80f83-17ca-4fa7-85b3-a6180f9eb81c.png)